extensions/mssql/src/features.ts

```diff
@@ -92,6 +103,34 @@ export class AccountFeature implements StaticFeature {
 		return params;
 	}
 
+	protected async refreshToken(request: contracts.RefreshTokenParams): Promise<contracts.TokenRefreshedParams> {
+
+		// find account
+		const accountList = await azdata.accounts.getAllAccounts();
+		const account = accountList.find(a => a.key.accountId === request.accountId);
+
+		// find tenant
+		const tenant = account.properties.tenants.find(tenant => tenant.id === request.tenantId);
+		if (!tenant) {
+			console.log(localizedConstants.failedToFindTenants);
```

Comment:
There aren't multiple tenants being searched for so this doesn't make sense. I would suggest adding in the ID too for debugging purposes. 

(and also make sure to include information about where this error came from when logging anything to the console)

`Failed to find tenant '${request.tenantId}' in account '${account.displayInfo.displayName}' when refreshing security token`

(make sure you're using parameters to the localize function for the values above, I just inlined them for clarity)